### ISSUE TITLE
Misleading add recipe command under Recipe Management mode UG

### ISSUE THREAD

**Issue by nickczh** [nickczh/pe#13]
![image.png](https://raw.githubusercontent.com/nickczh/pe/main/files/f03c8a50-874e-4c9d-a2d9-5ecd37a1e20b.png)

From this image, if I input the following:

![image.png](https://raw.githubusercontent.com/nickczh/pe/main/files/15a5f17d-25dc-49b9-af21-89514a1d9efe.png)

It still prompts me to input the recipe title again, making the UG unreliable because the command format is redundant.

**Comment by nus-se-script** [comment]
# Team's Response
![image.png](https://raw.githubusercontent.com/nus-cs2113-AY2324S2/pe/master/files/a02acb36-d898-4292-884a-621919cc5ce5.png)

Single Command:
![image.png](https://raw.githubusercontent.com/nus-cs2113-AY2324S2/pe/master/files/75bf039f-97b5-47e3-93c0-4f0833a8c051.png)

Separate Command:
![image.png](https://raw.githubusercontent.com/nus-cs2113-AY2324S2/pe/master/files/9d3c63d6-81d3-475a-814f-a04935371a99.png)

It is stated in the UG for recipe management mode that even though the command word is "add", "add something" will still be accepted for the convenience of the user.

In the UG, for singular commands, are written in one whole code while for the mentioned commands it is separate code.
Basically “command something” means you have to type together. “Command” “something” means you have to type the command first then it will prompt u to type the next part. It was done intentionally as in recipe mode, the user just need to type "add" and will be prompted for the "keyword".
# Items for the Tester to Verify
## :question: Issue response

Team chose [`response.Rejected`]

- [ ] I disagree

**Reason for disagreement:** [replace this with your explanation]

<catcher-end-of-segment><hr>
